src/networking/payloads/introductionrequestpayload.rs

```diff
@@ -32,125 +34,70 @@ struct IntroductionRequestPayload {
   identifier: u16,
 
   /// is a string that can be used to piggyback extra information.
-  extra_bytes: Vec<u8>,
+  extra_bytes: RawEnd,
 }
 
-impl Ipv8Payload for IntroductionRequestPayload {
```

Comment:
Indeed, the most valuable programming skill is the ability to *delete* code :wink: